### DISCUSSION TITLE
Copilot Chat not Loading after VS update to 17.7.0

### DISCUSSION THREAD

**Discussion by xxReinhart** [community/community#63546] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

After updating VS to version 17.1.0 Copilot Chat fails to load with the error file not found.  Here's the relevant line in ActivityLog.xml:

  <entry>
    <record>168</record>
    <time>2023/08/12 03:17:00.347</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>SetSite failed for package [ConversationsUIPackage]Source: &apos;Microsoft.VisualStudio.Conversations.UI&apos; Description: Could not load file or assembly &apos;Conversations.Shared, Version=0.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. The system cannot find the file specified.&#x000D;&#x000A;System.IO.FileNotFoundException: Could not load file or assembly &apos;Conversations.Shared, Version=0.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. The system cannot find the file specified.&#x000D;&#x000A;File name: &apos;Conversations.Shared, Version=0.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos;&#x000D;&#x000A;   at Microsoft.VisualStudio.Conversations.UI.Internal.ConversationsUIPackage.&lt;InitializeAsync&gt;d__5.MoveNext()&#x000D;&#x000A;   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine](TStateMachine&amp; stateMachine)&#x000D;&#x000A;   at Microsoft.VisualStudio.Conversations.UI.Internal.ConversationsUIPackage.InitializeAsync(CancellationToken cancellationToken, IProgress`1 progress)&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.AsyncPackage.&lt;&gt;c__DisplayClass21_0.&lt;&lt;Microsoft-VisualStudio-Shell-Interop-IAsyncLoadablePackageInitialize-Initialize&gt;b__1&gt;d.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.RethrowException(AggregateException e)&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.InternalGetResult(Boolean ignoreUIThreadCheck)&#x000D;&#x000A;&#x000D;&#x000A;WRN: Assembly binding logging is turned OFF.&#x000D;&#x000A;To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.&#x000D;&#x000A;Note: There is some performance penalty associated with assembly bind failure logging.&#x000D;&#x000A;To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].&#x000D;&#x000A;</description>
    <guid>{B1B98B90-307B-42EC-8931-03D92F5E13CD}</guid>
    <hr>0x80070002</hr>
    <errorinfo></errorinfo>
  </entry>

**Comment by xxReinhart** [comment, marked as answer]
Now works after Disabling the CoPilot Chat extension, closing VS then enabling CoPilot Chat, closing VS then on opening VS the Copilot Chat was available under View->Gethub CoPilot Chat selection.

**Reply by Jaimi** [reply]
Thanks! Both extensions quit loading for me today, and disabling/restarting/enabling/restarting fixed it for me also.

**Reply by PhLacoude** [reply]
Fixed it for me as well. Thank you!
